From c71a0bf1fdd48a97be1524f2d450575a590c07db Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Tue, 7 Mar 2017 17:37:30 +0800
Subject: [PATCH] arm64: dts: rockchip: enable mbox related for rk3368-tb

This adds enable mbox and scpi function for rk3368-tb board.

Change-Id: I0b4866b72ad59892bc1c051c60bafd1a15ddce14
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
index 58e12327481b..aaf245b3d96f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
@@ -636,6 +636,14 @@
 	status = "disabled";
 };
 
+&mailbox {
+	status = "okay";
+};
+
+&mailbox_scpi {
+	status = "okay";
+};
+
 &rga {
 	status = "okay";
 };
-- 
2.35.3

